### PR TITLE
Update mobile version to desktop

### PR DESCRIPTION
**Pull request by george827**
### Milestone: update mobile responsiveness for the desktop version

- change the layout for desktop
- Change images 
- Add navigation menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,279 +1,292 @@
...
+                  <li> <a href="#"><img src="img/Vectorm.png" alt="m"></a> </li>
+                  <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
+                  <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
+               </ul>
             </div>
-        </section>
-    
-    </article>
-
-    <section id="main-works">
```
Comment (aimalamiri) [inline index.html:55]: Please make sure to put some space between two sections.

image

File: index.html [hunk 2]
```diff
@@ -1,279 +1,292 @@
...
-                        <li> <a href="#" class="btn"> javascript </a></li>
-                    </ul>
-                </div>
-                <a href="#"> <div class="btn-main">See Project</div> </a> 
-                
+            <section class="main" id="snap-item">
+               <div class="head-one-alt">
+                  <h2> Uber Navigation
+                  </h2>
+               </div>
```
Comment (aimalamiri) [inline index.html:166]: Looks great, but please make sure to follow the exact Figma design, and some padding to the left.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @george827, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The Desktop version is designed very well and look fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:166]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @george827    ,
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**